src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -204,7 +204,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             _optionCompareText = optionCompareText
             _embedVbCoreRuntime = embedVbCoreRuntime
             _suppressEmbeddedDeclarations = suppressEmbeddedDeclarations
-            _parseOptions = parseOptions
+            _parseOptions = If(parseOptions, VisualBasicParseOptions.Default) ' WHY: 
```

Comment:
This is an unrelated and unwanted public API behavior change.

---
In reply to: [72994762](https://github.com/dotnet/roslyn/pull/12629#discussion_r72994762) [](ancestors = 72994762)